RadarSDK/RadarLocationManager.m

```diff
@@ -223,7 +242,7 @@ - (void)startUpdates:(int)interval {
             [self.timer invalidate];
         }
 
-        self.timer = [NSTimer scheduledTimerWithTimeInterval:interval
+        self.timer = [NSTimer scheduledTimerWithTimeInterval:(interval + 0.1)
```

Comment:
Gonna pull this into a separate PR — doing this b/c I'm seeing lots of 9.95 intervals skipped